src/app/app-platform/ContentApp.h

```diff
@@ -94,11 +96,30 @@ class ContentAppClientCommandSender
 class DLL_EXPORT ContentApp
 {
 public:
+
+    struct SupportedCluster {
+        int clusterIdentifier { -1 };
+        int features { -1 };
```

Comment:
Again, how is this supposed to deal with a featuremap that actually uses bit 31?

Reply:
Is there a type alias for the features map as well similar to ClusterId/AttributeId/CommandId or is it just uint32_t?